package.json

```diff
@@ -8,8 +8,6 @@
   },
   "scripts": {
     "doctoc": "doctoc README.md",
-    "format": "prettier --write .",
```

Comment:
There is a bunch of CI scripts that rely on these to be present, I quickly found at least
https://github.com/ory/ci/blob/efcf157174048909dacafa29463f95aba629e37c/src/scripts/changelog/generate.sh#L52
We have to ensure this is not required anymore by any automation.

Reply:
Great catch! I'll fix those as part of these PRs.